### PR TITLE
feat: add "malva" and "markup" dprint plugins

### LINKED ISSUE #16: Support svelte

### LINKED ISSUE #65: Does it support 3rd-party dprint plugins?
It seems that this just supports official dprint plugins. How can we add 3rd-party plugins to it? (For example, to format CSS and/or HTML/Vue.)

### PR DESCRIPTION
**Pull request by g-plane**
Close #16 
Close #65

### CODE REVIEW

**Comment by g-plane** [comment]
@so1ve

**Comment by so1ve** [comment]
Thanks! (Not sure why this PR is not in my GitHub notifications 😅)

**Comment by g-plane** [comment]
Failures are caused by Node.js ESM...

**Comment by so1ve** [comment]
I'm working on this.

**Comment by so1ve** [comment]
@g-plane Could you please add CJS versions to these packages?

**Comment by g-plane** [comment]
But, I want to keep pure ESM. It looks that currently CI fails about installation, not modules interop.

**Comment by so1ve** [comment]
I've updated lockfile and CI is still failing. Classic ESLint config only supports CommonJS so this plugin cannot migrate to pure ESM, thus I can't use pure-ESM dependencies (I downgraded `find-up` for this reason)

**Comment by g-plane** [comment]
However, I don't want to maintain two kinds of module format at this moment. It sounds that adding CJS support then removing them in the future can waste time.

**Comment by g-plane** [comment]
File path can be incorrect after bundling. Should we import WebAssembly files manually?

**Comment by so1ve** [comment]
> File path can be incorrect after bundling

And this is the reason why I gave up this path

> I don't want to maintain two kinds of module format

You may replace `tsc` with `pkgroll`. Still zero-config but supports dual package

**Comment by g-plane** [comment]
Can we use `require.resolve`? It should avoid the problems about bundling and Node.js ESM.

**Comment by so1ve** [comment]
Got it, I'll use my custom logic to resolve these `.wasm` files

**Comment by g-plane** [comment]
I will implement it in this PR.

File: src/format.ts [hunk 1]
```diff
@@ -65,12 +64,13 @@ export class Formatter {
 			pluginConfig.dockerfile,
 		);
 		this.malva = createFormatter(
-			getMalvaPath(),
+			join(require.resolve("dprint-plugin-malva"), "..", "./plugin.wasm"),
```
Comment (g-plane) [inline src/format.ts:67]: I think `path.dirname` is better than `..`.

**Comment by so1ve** [comment]
Windows tests are failing...

**Comment by g-plane** [comment]
Sorry I don't have Windows, but I will try fixing it.

**Comment by so1ve** [comment]
Anyway, I plan to move to `eslint.RuleTester`. I'll fix the ci issues

**Review by so1ve — APPROVED** [review]

**Pull request merged**